returnn/datasets/audio.py

```diff
@@ -410,8 +430,16 @@ def get_data_shape(self, key):
         if key == "data" and self.feature_extractor is not None:
             if self.feature_extractor.num_channels is not None:
                 return [self.feature_extractor.num_channels, self.feature_extractor.get_feature_dimension()]
+        if key == "raw":
+            return []
         return super(OggZipDataset, self).get_data_shape(key)
 
+    def is_data_sparse(self, key) -> bool:
+        """:return: whether data entry with `key` is sparse"""
+        if key == "raw":
+            return False
+        return super().is_data_sparse(key)
```

Comment:
Why do you rely on `super().is_data_sparse()` here? I would simply return it directly for all the possible cases.